### PR TITLE
Added example for Annulus

### PR DESCRIPTION
**Pull request by JeroSQ**
## Changelog / Overview

Added example for Annulus in response to Issue #1519 

## Explanation for Changes

[Documentation Reference](https://manimce--1599.org.readthedocs.build)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [x] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: manim/mobject/geometry.py [hunk 1]
```diff
@@ -767,6 +767,30 @@ def __init__(self, outer_radius=1, inner_radius=0, **kwargs):
 
 
 class Annulus(Circle):
+    """Region between two concentric :class:`Circle`s.
+
+    Parameters
+    ----------
```
Comment (Nikhil0504) [inline manim/mobject/geometry.py:773]: Make sure to add the typings in the function and not the documentation part.
See: https://docs.manim.community/en/latest/contributing/typings.html

**Comment by JeroSQ** [comment]
PEP8 will kill me one day

**Review by Nikhil0504 — CHANGES_REQUESTED** [review]
Requested some changes

File: manim/mobject/geometry.py (hunk 1 above)
Comment (JeroSQ) [reply manim/mobject/geometry.py:773]: I'm sorry I don't understand. I just did what I saw in other examples

Comment (Nikhil0504) [reply manim/mobject/geometry.py:773]: You are supposed to remove inner_radius ~: :class:`float`, optional~ 
and put it under the function parameters like this:
`inner_radius: Optional[float] = 1,`

Go through the link once again to get an idea on what typings to use where.

File: manim/mobject/geometry.py [hunk 2]
```diff
@@ -771,9 +771,9 @@ class Annulus(Circle):
 
     Parameters
     ----------
-    inner_radius : Optional[float] = 1
+    inner_radius : Optional[float] = 1,
```
Comment (Nikhil0504) [inline manim/mobject/geometry.py:774]: You are supposed to do it in line 796 and 797

**Review by Nikhil0504 — APPROVED** [review]
LGTM for now, more changes can be added later on. Thank you for the contribution.

**Comment by JeroSQ** [comment]
Thanks to you for reviewing the PR and telling me what was wrong with it

File: manim/mobject/geometry.py [hunk 3]
```diff
@@ -757,10 +757,34 @@ def __init__(self, outer_radius=1, inner_radius=0, **kwargs):
 
 
 class Annulus(Circle):
+    """Region between two concentric :class:`Circle`s.
```
Comment (behackl) [inline manim/mobject/geometry.py:760]: This unfortunately doesn't render correctly, ReST doesn't really allow letters directly following links:

image

Here is a potential workaround:

```suggestion
    """Region between two concentric :class:`Circles <.Circle>`.
```

**Review by behackl — CHANGES_REQUESTED** [review]

File: manim/mobject/geometry.py (hunk 3 above)
Comment (JeroSQ) [reply manim/mobject/geometry.py:760]: Oh, sorry

**Review by behackl — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**